README.md

```diff
@@ -27,7 +27,7 @@ Usage: https-proxy <options>
 
 Options:
   -t, --target  target address, like http://localhost:80  [required]
-  -p, --port    port to use for https                     [required]
+  -p, --port    port to use for https, default is 443
```

Comment:
IMHO the most likely reasonable default is a 4 digit port, because this util is mostly used locally.
So something like 8080 or 3000 would be a good default